### ISSUE TITLE
Question: Kodi 18 Branch

### ISSUE THREAD

**Issue by OnceUponALoop** [aqw/xbmc-gamepass#412]
Will you be creating a separate branch for the kodi 18 development?

**Comment by aqw** [comment]
Hey @OnceUponALoop This is a duplicate of #373 

But to answer your question directly: there will likely be a branch for Kodi 18 created, though when I cannot say for sure. The update to Kodi 18 requires more changes than usual (not the least of which is Python 3 compatibility).

As a quick overview of the roadmap: The ongoing work in the pigskin repo is address these fixes (among other things). Once the pigskin work is done, then I will focus on updating this addon's code. That code will work for both Kodi 17 and 18. /Then/ I will create a branch for Kodi 18 that makes the backward incompatible skin changes that Kodi 18 will require.
